nixos/doc/manual/man-nixos-rebuild.xml

```diff
@@ -315,6 +319,27 @@ $ ./result/bin/run-*-vm
      </para>
     </listitem>
    </varlistentry>
+   <varlistentry>
+     <term>
+       <option>--builders</option>
+       <replaceable>builder-spec</replaceable>
+     </term>
+     <listitem>
+       <para>
+         Allow to specify remote builders ad-hoc for building the new system.
+         This requires user executing <command>nixos-rebuild</command> (usually
+         root) to be configured as trusted user in nix daemon. This can be
+         achived by using the <literal>nix.trustedUsers</literal> nixos option.
```

Comment:
nixos options probably also have a special markup?

Reply:
`as trusted user in nix daemon` -> `as a trusted user in the Nix daemon`.

`nixos option` -> `NixOS option`.